mosec/server.py

```diff
@@ -63,12 +57,32 @@ def __init__(
         signal.signal(signal.SIGTERM, self._terminate)
         signal.signal(signal.SIGINT, self._terminate)
 
-    def _validate(self):
+    def _validate_server(self):
         assert len(self._worker_cls) > 0, (
             "no worker registered\n"
             "help: use `.append_worker(...)` to register at least one worker"
         )
 
+    @staticmethod
+    def _validate_arguments(
+        worker,
+        num,
+        max_batch_size,
+        start_method,
+    ):
+        def at_least_one_int(num, name):
+            assert isinstance(
+                num, int
+            ), f"{name} must be integer but you give {type(num)}"
```

Comment:
```py
isinstance(True, int)
>>> True
```

Or maybe you can try to convert with `int(num)` and cache the exception?

The function name is a little confusing. I thought it restricts a list to have at least one integer.

Reply:
The code is interesting...
Okay will modify this part!